### ISSUE TITLE
Document Formatting w/ Keyboard or Markdown

### ISSUE THREAD

**Issue by thetrumper** [indentlabs/notebook#351]
I'd like to suggest adding a feature of easier formatting when creating documents on notebook.

For example, if I go to create a new document rather than having to highlight text and then apply bolding it'd be easier if I could use key commands like Ctrl + B or use Markdown to format the documents. 

As it stands now, when using markdown in a document it will sometimes apply the formatting when hovering over the document but actually viewing the document (NOT EDITING) just shows the markdown formatting. 

In other words, it'd just be nice to have easier ways of formatting instead of having to select text and choose from the menu.

**Comment by drusepth** [comment]
This is a good suggestion. 

Quick question for you -- when you highlight text and press Ctrl + B to bold the text, it doesn't already bold that text for you? (Similarly, you can't press Ctrl + B to start typing and have it be bold text already?) This seems like a bug that needs fixed, as this is intended and should already be implemented. May I ask what browser you're using so I can test and fix there? Bolding, italics, and underlining should all work with keyboard commands.

**Comment by thetrumper** [comment]
I just tested it again and it worked with keyboard commands. It might have been just a glitch with my keyboard because I swear it wasn't working when I was trying to write earlier. For reference, though, I am using Chrome.

Markdown text still does not show up properly when viewing, however, but I'm not sure that that was implemented yet.

https://imgur.com/a/X2NPBj3
There's an image of what it shows up like in the "hover box" as I like to call it and then when I actually go to view it.

**Comment by drusepth** [comment]
Ah, thanks for the screenshot. At first I was hesitant to simultaneously support markdown alongside HTML in documents because I wasn't sure if there'd ever be any conflicts in the preview (are there times where unintentional markdown would make the preview look strange to a normal user if they didn't know what markdown was and just happened to have valid markdown in their document?), but I think this is fine; probably better to support whatever the users want to use.

I'm gonna go ahead and rename this issue to narrow the scope to properly previewing markdown in documents since the keyboard commands seem to be working. :+1:
